File: Vanilla/sources/zomboid/zombie/radio/devices/PresetEntry.java
package zombie.radio.devices;

public final class PresetEntry {
   public String name = "New preset";
   public int frequency = 93200;

   public PresetEntry() {
   }

   public PresetEntry(String var1, int var2) {
      this.name = var1;
      this.frequency = var2;
   }

   public String getName() {
      return this.name;
   }

   public void setName(String var1) {
      this.name = var1;
   }

   public int getFrequency() {
      return this.frequency;
   }

   public void setFrequency(int var1) {
      this.frequency = var1;
   }
}

File: Vanilla/sources/zdoc/media/lua/shared/Library/PresetEntry.lua
---@class PresetEntry : zombie.radio.devices.PresetEntry
---@field public name String
---@field public frequency int
PresetEntry = {}

---@public
---@return int
function PresetEntry:getFrequency() end

---@public
---@param arg0 int
---@return void
function PresetEntry:setFrequency(arg0) end

---@public
---@return String
function PresetEntry:getName() end

---@public
---@param arg0 String
---@return void
function PresetEntry:setName(arg0) end
